src/functions-reference/matrix_operations.Rmd

```diff
@@ -1611,3 +1612,16 @@ Number of components of v less than v[s]
 `int` **`rank`**`(row_vector v, int s)`<br>\newline
 Number of components of v less than v[s]
 
+## Reverse Functions {#reverse-functions}
+
+<!-- vector; reverse; (vector v); -->
+\index{{\tt \bfseries reverse }!{\tt (vector v): vector}|hyperpage}
+
+`vector` **`reverse`**`(vector v)`<br>\newline
+Create a copy of the vector with the elements in reverse order.
```

Comment:
This isn't a copy any more than sorting is a copy.  It returns a new vector containing the elements of the argument vector in reverse order.